core/command/engineErrors.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  execute: function (config) {
```

Comment:
You can just export the anon function here.  Skip the execute property and the object all together.